Polyhedron/demo/Polyhedron/Plugins/PMP/Detect_sharp_corners_plugin.cpp

```diff
@@ -0,0 +1,152 @@
+#include <QApplication>
```

Comment:
please do not add another plugin, and simply enrich the existing `Detect_sharp_edges_plugin.cpp` as discussed by email